src/main/java/org/apache/rocketmq/dashboard/service/impl/DlqMessageServiceImpl.java

```diff
@@ -65,4 +68,16 @@ public MessagePage queryDlqMessageByPage(MessageQuery query) {
         }
         return messageService.queryMessageByPage(query);
     }
+
+    @Override
+    public ConsumeMessageDirectlyResult batchResendDlqMessage(List<DlqMessageRequest> dlqMessages) {
+        List<ConsumeMessageDirectlyResult> consumeMessageDirectlyResults = new LinkedList<>();
+        for (DlqMessageRequest dlqMessage : dlqMessages) {
+            ConsumeMessageDirectlyResult consumeMessageDirectlyResult = messageService.consumeMessageDirectly(dlqMessage.getTopicName(),
+                dlqMessage.getMsgId(), dlqMessage.getConsumerGroup(),
+                dlqMessage.getClientId());
+            consumeMessageDirectlyResults.add(consumeMessageDirectlyResult);
+        }
+        return consumeMessageDirectlyResults.get(0);
```

Comment:
Batch resend the message but just get the first result. If we do a batch operation, we should get all the results.